### ISSUE TITLE
Inconsistent logging

### ISSUE THREAD

**Issue by pulkitsinghal** [ShoppinPal/sp-json-logger#14]
If I use `message` alone:
```
logger.tag('Client File').debug({ message: `clientFile: ${clientFile}` });
```
then in kibana, the logs show up in the `message` field after aggregation

If I use `message` with other properties:
```
logger.debug({ messageId: messageId, commandName: commandName, message: `Saving total line items ${rows.length}` });
```
then in kibana, the logs show up in the `log.message` field

Can you please fix the code so that its consistent?

**Comment by pulkitsinghal** [comment]
cc @harshadyeola - i logged it here

**Comment by yogesh8177** [comment]
Thanks for pointing that out.

Looks like from `sp-json-logger v2.0.0`, `message` property is a child of `log` object.  

Will verify it and let you know if this is the cause of the problem.

**Comment by yogesh8177** [comment]
Yes, it's version issue, all the logs for version 1 had separate `message` property, newer logs (starting with v2.0.0) will follow `log.message` format from here onwards.

**Comment by yogesh8177** [comment]
I think the issue is something else, because the above two syntax works properly by outputting in `log. message` format.

**Comment by yogesh8177** [comment]
Versions won't cause the issue as we were already using `log.message` format. I checked your syntax and could see that logs are getting printed as `log.message` in both the cases. 
So logs are printing fine, problem might lie after logs are pushed to elk.

Thus, I don't think I need to change any code as of now!

**Comment by yogesh8177** [comment]
Regarding `message` property appearing as a candidate to aggregate, it belongs to error logs.

If someone did 
`logger.error({ message: "some message", err: error });`
Then the `message` field is for such logs. It does not get encapsulated inside `err` as it might have its own message property which we don't want to override
